packages/react-email/src/cli/utils/preview/hot-reloading/get-imported-modules.ts

```diff
@@ -0,0 +1,32 @@
+import { parse } from '@babel/parser';
```

Comment:
I first did this with a regex, but it ended up having weird behavior on a few edge cases. So, what I ended up doing, was use babel to parse the JavaScript files into their AST, to then walk through it to find the imports/requires.